js/jquery.dvfCharts.js

```diff
@@ -100,7 +100,6 @@
       }
 
       data.type = this.options.chart.data.type;
-      data.labels = this.options.chart.data.labels.show;
```

Comment:
if this needed to be a bool could have maybe just done
```
data.labels = (this.options.chart.data.labels.show ? true : false);
```

Reply:
Thanks @jez500 - updated!